internal/provider/provider.go

```diff
@@ -63,6 +71,7 @@ func configure(version string, p *schema.Provider) func(context.Context, *schema
 		baseURL := d.Get("base_url").(string)
 		concurrency := d.Get("concurrency").(int)
 		organization := d.Get("organization").(string)
+		enterprise := d.Get("enterprise").(string)
```

Comment:
Is it worth ensuring only one of enterprise or organisation is set?

Reply:
It is enforced by Terraform provider schema: https://github.com/form3tech-oss/terraform-provider-githubipallowlist/blob/d2218ceafd2652e183fe003791530db62f00dbd3/internal/provider/provider.go#L29